README.md

```diff
@@ -59,11 +59,6 @@ spaCy integration for curated transformers is provided by the
 [`spacy-curated-transformers`](https://github.com/explosion/spacy-curated-transformers)
 package.
 
-## ⚠️ Warning: Tech Preview
```

Comment:
We should keep this until the release next week. This is the landing page that people will see, even if this is the `main` branch.

Reply:
Reverted.